ballista/rust/core/src/serde/physical_plan/mod.rs

```diff
@@ -235,7 +235,11 @@ impl AsExecutionPlan for PhysicalPlanNode {
             PhysicalPlanType::GlobalLimit(limit) => {
                 let input: Arc<dyn ExecutionPlan> =
                     into_physical_plan!(limit.input, registry, runtime, extension_codec)?;
-                Ok(Arc::new(GlobalLimitExec::new(input, limit.limit as usize)))
+                Ok(Arc::new(GlobalLimitExec::new(
```

Comment:
Seems like here we should just update the protobuf struct to hold both skip and fetch. In our downstream I just used `0` to encode `None`. I _think_ that should work unless having `fetch = Some(0)` is meaningful in some scenario

Reply:
See https://github.com/coralogix/arrow-ballista/commit/68aecb8e7e16ce7346dac3192153e000d3f0dfe4

Reply:
Actually there was a bug in that version :) See the fix here https://github.com/coralogix/arrow-ballista/pull/1/commits/830b2d58a13710bb3e28337be5a641716155487e